media-gfx/qimgv/qimgv-0.8.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Old copyright header which makes me believe you didn't commit this with repoman like your git commit message makes it believe...

Reply:
Oh, I didn't know you were supposed to commit with repoman. I didn't even know you could commit with repoman haha.
I just had it in my commit message because it was stated in the commit message formatting

Reply:
Using repoman to commit is optional, but couraged. The line in your commit message, `Package-Manager: Portage-2.3.69, Repoman-2.3.16` usually just implies it was made with repoman ;) you shouldn't write this automatically.